src/components/RegionSettings.tsx

```diff
@@ -70,15 +95,18 @@ export default function RegionSettings() {
         {regions &&
           regions.map((region) => {
             return (
-              <Card style={{ margin: 10 }} variant="outlined">
+              <Card style={{ marginBottom: 10 }} variant="outlined">
```

Comment:
Trochę źle jest mieszać style `inline` oraz klasy `css`, ponieważ to zmiejsza czytelnośc kodu, z racji tego, że musisz sprawdzać dwa miejsca żeby wiedzieć jak dokładnie wyglada komponent.
Ja bym wyniósł to do klasy.